src/components/Jobs/Card/Card.js

```diff
@@ -0,0 +1,56 @@
+import PropTypes from 'prop-types'
+import './card.css'
+
+function Card({ jobData }) {
+  let date = ''
+  try {
+    date = new Date(jobData.created).toLocaleDateString()
+  } catch (err) {
+    console.error('Invalid date.', jobData.created)
+  }
+
+  function removeHTML(text) {
+    // this regex removes html tags
+    return text.replace(/<[^>]*(>|$)|&nbsp;|&zwnj;|&raquo;|&laquo;|&bull;|&gt;/g, '')
+  }
+
+  return (
+    <div className="grid-container">
+      <div className="grid-item grid-item-1">{removeHTML(jobData.title)}</div>
+      <div className="grid-item grid-item-2">
+        Company: {removeHTML(jobData.company.display_name)} -{' '}
+        {removeHTML(jobData.location.display_name)}
+      </div>
+      <div className="grid-item grid-item-3">
+        Remote:
+        {jobData.description.toLowerCase().indexOf('remote') > -1 ||
+        jobData.title.toLowerCase().indexOf('remote') > -1 ||
+        jobData.title.toLowerCase().indexOf('work from home') > -1 ||
+        jobData.description.toLowerCase().indexOf('work from home') > -1
+          ? ' Yes'
+          : ' No'}
```

Comment:
Nested ternary makes it hardly readable.

Reply:
done